spec/fixtures/flavors/rmarkdown.ass

```diff
@@ -555,3 +555,84 @@ text.md {
     }
   }
 }
+
+@14
+"```{foo}" {
+  text.md {
+    fenced.code.md {
+      "```": punctuation.md
+      "{": punctuation.md
+      "foo": language.constant.md
+      "}": punctuation.md
+    }
+  }
+}
+
+@15
+"```{js label, test=1, engine="2"}" {
+  text.md {
+    fenced.code.md {
+      "```": punctuation.md
+      "{": punctuation.md
+      "js": language.constant.md
+      " "
+      "label": label.string.md
+      ", test=1, engine="2""
+      "}": punctuation.md
+    }
+  }
+}
+
+@16
+"```{r label }" {
+  text.md {
+    fenced.code.md {
+      "```": punctuation.md
+      "{": punctuation.md
+      "r": language.constant.md
+      " "
+      "label": label.string.md
+      " "
+      "}": punctuation.md
+    }
+  }
+}   
+
+@17
+"```{js nolabel= "nolabel", engine="first"  ,   engine ="second"   , engine="third"}" {
+  text.md {
+    fenced.code.md {
+      "```": punctuation.md
+      "{": punctuation.md
+      "js": language.constant.md
+      " nolabel= "nolabel", engine="first"  ,   engine ="second"   , engine="third""
+      "}": punctuation.md
+    }
+  }
+}
+
+@18
+"```{}"{
```

Comment:
I'm not sure if an empty element should be recognized as fenced-code. I don't think there's an explicit reference to empty parameters in [this spec sheet](https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf), but since it is empty, it has no meaning, and highlighting it as a valid code-block might confuse users.